modules/administration/pages/tls-certs-monitoring.adoc

```diff
@@ -0,0 +1,88 @@
+[[tls-certs-monitoring]]
+= TLS Certificates for Monitoring
+
+By default, {productname} does not provide any certificates for securing monitoring configuration.
+For providing security, you can import custom certificates, self-signed or signed by a third party certificate authority (CA).
+
+This section demonstrates how to generate client/server certificates for Prometheus and Node exporter minions self-signed with SUSE Manager CA.
+
+The host name of the TLS keys and certificates must match the fully qualified host name of the machine you deploy them on.
+You can set the host names in the ``X509v3 Subject Alternative Name`` section of the certificate.
+You can also list multiple host names if your environment requires it.
+
+
+== Generate client/server keys and certificates for monitoring
+
+To generate TLS keys and certificates for use with Prometheus server and node exporters you can follow these instructions:
+
+.Procedure: Create TLS keys and certificates
+
+. On the {productname} Server, change to the ``ssl-build`` directory:
++
+----
+cd /root/ssl-build
+----
+. Create private key:
++
+----
+openssl genrsa -out server.key 2048
+----
+. Create Certificate Signing Request (CSR) using the private Key.
+  An example OpenSSL configuration can be found below.
+  Please ensure that the set-cname parameter is the fully qualified domain name of your SUSE Manager Server.
++
+----
```

Comment:
I think we should add backslashes at the end a line if the command with the parameters continues with the next line.